ParkingGarage/Program.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+					
+public class Program
+{
+	public static void Main()
+	{
+		Person chase = new Person("Chase");
+		Car blueCar = new Car("blue", chase);
+		Garage smallGarage = new Garage(2);
+		
+		smallGarage.ParkCar(blueCar, 0);
+		Console.WriteLine(smallGarage.Cars);
+	}
+}
+
+public class Person
+{
+    public string name;
+
+	public Person(string initialName)
+	{
+		this.name = initialName;
+	}
+}
+
+class Car
+{
+    public Car(string initialColor, Person initialPerson)
+    {
+    	Color = initialColor;
+		this.person = initialPerson;
+		
+    }
+    
+    public string Color { get; private set; }
+	public Person person;
+}
+
+
+class Garage
+{
+    private Car[] cars;
+    
+    public Garage(int initialSize)
+    {
+    	Size = initialSize;
+	    this.cars = new Car[initialSize];
+    }
+    
+    public int Size { get; private set; }
+    
+    public void ParkCar (Car car, int spot)
+    {
+        cars[spot] = car;
+    }
+    
```

Comment:
I was expecting a method to remove a car for a parking garage, and a method to add a person to car.